src/ListView/ListView.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+/**
+ * ListView Component for Patternfly React
+ */
+const ListView = ({ children, listViewClass }) => (
+  <div className={listViewClass}>
```

Comment:
Or you're saying className={classNames('list-pf', className)} would produce the same?